tests/lib/runner.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  isSetup: false,
+  setup () {
+    this.isSetup = true
+    this.tests = []
+    this.errors = []
+    this.failures = 0
```

Comment:
you don't need to count the failures manually, you have this value in `this.errors.length`